pkg/util/clusteragent/clusteragent_test.go

```diff
@@ -2,6 +2,7 @@
 // under the Apache License Version 2.0.
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-2019 Datadog, Inc.
+// +build !windows
```

Comment:
Although it does not run (yet?), this code is shipped in the windows build of the agent. What is the rationale for disabling the tests? I don't see any platform-specific code in them

Reply:
@xvello the tests were failing.  My plan was to get the relevant code fixed, and file cards for things like this.  Feel free to fix ;)